core/instance.go

```diff
@@ -917,3 +920,9 @@ func min(x, y int) int {
 	}
 	return y
 }
+func max(x, y int) int {
```

Comment:
do we need to define this again? I think we had it defined somewhere else.

Hopefully we'll soon get some sort of generics support in Go to avoid the need to define this sort of things.

Reply:
I only saw `min` defined in this way, right above where I added this change.

Reply:
Okay, thanks